### PR TITLE
Add Conv2dSubsampling1 module and test it in AphasiaBank ASR recipe

### PR DESCRIPTION
**Pull request by tjysdsg**
`Conv2dSubsampling1` module is similar to `Conv2dSubsampling` except the subsampling ratio is 1:1.

AphasiaBank English ASR experiments using E-Branchformer+WavLM showed a CER decrease from 17.7 to 17.0 by switching from `Conv2dSubsampling2` to `Conv2dSubsampling1`.

The user can specify `input_layer: conv2d1` in the encoder configuration to use this module.

### CODE REVIEW

**Review by simpleoier — APPROVED** [review]
Better results!
Looks good to me in general!
Just one concern about the name of the new module, `Conv2dSubsampling1` doesn't do subsampling actually.

**Comment by tjysdsg** [comment]
> Better results! Looks good to me in general! Just one concern about the name of the new module, `Conv2dSubsampling1` doesn't do subsampling actually.

Oh right, what's your suggestion? Should we just call it `Conv2d1` or `Conv2dInputLayer`?

File: egs2/aphasiabank/asr1/README.md [hunk 1]
```diff
@@ -9,6 +9,24 @@
 - Git hash: `39c1ec0509904f16ac36d25efc971e2a94ff781f`
     - Commit date: `Wed Dec 21 12:50:18 2022 -0500`
 
+## asr_train_asr_ebranchformer_small_wavlm_large1
+
+- [train_asr_ebranchformer_small_wavlm_large.yaml](conf/tuning/train_asr_ebranchformer_small_wavlm_large.yaml)
```
Comment (pyf98) [inline egs2/aphasiabank/asr1/README.md:14]: Can you add a short note here to explain the difference? E.g., this config uses xxx as the input layer which does not perform downsampling.

Comment (pyf98) [reply egs2/aphasiabank/asr1/README.md:14]: Is this config correct? It looks same as the previous one.

Comment (tjysdsg) [reply egs2/aphasiabank/asr1/README.md]: Thanks! Fixed in 4734d75

**Comment by sw005320** [comment]
> > Better results! Looks good to me in general! Just one concern about the name of the new module, `Conv2dSubsampling1` doesn't do subsampling actually.
> 
> Oh right, what's your suggestion? Should we just call it `Conv2d1` or `Conv2dInputLayer`?

I think `Conv2dSubsampling1` is OK, but it is better to add a comment that this does not do any subsampling in appropriate places.

**Comment by tjysdsg** [comment]
@sw005320 I made the comment clearer about the functionality in c1dc65f

**Comment by sw005320** [comment]
Thanks a lot!

**Pull request merged**